test/tensors/test_native.py

```diff
@@ -0,0 +1,63 @@
+from syft.frameworks.torch.hook import TorchHook
+from syft.frameworks.torch.tensors import TorchTensor
+from syft.frameworks.torch.tensors import PointerTensor
+from syft.workers.virtual import VirtualWorker
+from unittest import TestCase
+import torch
+
+
+class TestTorchTensor(TestCase):
+    def test_owner_default(self):
+        hook = TorchHook()
+        tensor = TorchTensor()
+
+        assert tensor.owner == hook.local_worker
+
+    def test_create_pointer(self):
+        hook = TorchHook()
+        tensor = TorchTensor()
+        bob = VirtualWorker()
```

Comment:
for our other unit tests previously we were able to initialize one set of workers at the top and then use them for all tests (made them run a bit faster)

Reply:
A good way to do this would be to create a conftest.py and put all the worker initialization in there. That way we could use them across all tests